src/vulnerabilities/strategies/npm-audit.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable class-methods-use-this */
+"use strict";
+
+// Require Third-party Dependencies
+const Arborist = require("@npmcli/arborist");
+
+// CONSTANTS
+const { constants } = require("../../utils");
+const { VULN_MODE_NPM_AUDIT } = require("../strategies");
+
+
+function NPMAuditStrategy() {
```

Comment:
What do you think about using a name which contains the "what" of this function as `getNpmAuditStrategy` (just an example)? If I miss something there, let me know :) ^^

Reply:
Hello Tony, I understand your point. I didn't explicitly put the "what" because this function is more used as a constructor (without the "new" keyword) rather than a getter function (basically, there is no use case of retrieving directly the npm audit strategy, it is initialized under the hood by methods in the vulnerabilitiesSource.js) file. Let me know if that makes sense